### PR TITLE
2.3.11 release

### PR DESCRIPTION
**Pull request by 2m**

### CODE REVIEW

File: news/_posts/2015-05-19-akka-2.3.11-released.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+layout: news
+title: Akka 2.3.11 Released!
+author: Martynas Mickevičius
+short: Akka 2.3.11 Released
+---
+
+*Dear hAkkers,*
+
+We—the Akka committers—are pleased to be able to announce the availability of Akka 2.3.11. This is the eleventh maintenance release of the 2.3 branch. This release contains several important fixes:
```
Comment (patriknw) [inline news/_posts/2015-05-19-akka-2.3.11-released.md]: s/several/a few/ ?

**Comment by patriknw** [comment]
LGTM

File: _includes/download.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
       var activatorLink = $("#activator-link")
       activatorLink.prop("href", "http://downloads.typesafe.com/typesafe-activator/" + data.version + "/typesafe-activator-" + data.version + ".zip")
       activatorLink.text(activatorLink.text().replace("1.2.7", data.version));
-      activatorLink.text(activatorLink.text().replace("2.3.10", data.akkaVersion));
+      activatorLink.text(activatorLink.text().replace("2.3.11", data.akkaVersion));
```
Comment (rkuhn) [inline _includes/download.html]: no, this needs to be the version that is to be automatically replaced in the page, which is 2.3.9 (for whatever reason). Please also leave a comment, we broke and fixed this several times now ;-)

Comment (rkuhn) [inline _includes/download.html]: ah, no, it was fixed the other way around now — confusing … but then I also didn’t have breakfast yet ;-)

File: news/_posts/2015-05-19-akka-2.3.11-released.md (hunk 1 above)
Comment (rkuhn) [inline news/_posts/2015-05-19-akka-2.3.11-released.md]: “several” is more formally correct, “a few” sounds more informal; the quantitative information is rather vague in both cases

File: news/_posts/2015-05-19-akka-2.3.11-released.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+ - a fix for BatchingExecutor: https://github.com/akka/akka/issues/16327
```
Comment (rkuhn) [inline news/_posts/2015-05-19-akka-2.3.11-released.md]: `- a fix for BatchingExecutor ([#16327](<link here>); symptom is a requirement failure when using blocking{} in Future callbacks)`

File: news/_posts/2015-05-19-akka-2.3.11-released.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+ - a fix for a performance regression caused by a previous bugfix in IO: https://github.com/akka/akka/issues/17391
+ - a revert of a performance optimization in 2.3.9 that caused actor starvation when multiple dispatchers were involved (critical) https://github.com/akka/akka/issues/17341
```
Comment (rkuhn) [inline news/_posts/2015-05-19-akka-2.3.11-released.md]: also make these ticket links smaller as above

**Comment by rkuhn** [comment]
Thanks, LGTM after the minor additions.

File: _includes/download.html (hunk 2 above)
Comment (patriknw) [inline _includes/download.html]: yeah, I think whatever is written here is replaced unless the communication with activator api fails

File: news/_posts/2015-05-19-akka-2.3.11-released.md (hunk 1 above)
Comment (patriknw) [inline news/_posts/2015-05-19-akka-2.3.11-released.md]: ok, I thought several > a few

**Comment by 2m** [comment]
Thanks for the review. Feedback addressed. Merging.

**Pull request merged**